Tests/SwiftParserTest/translated/RecoveryTests.swift

```diff
@@ -2174,15 +2174,17 @@ final class RecoveryTests: XCTestCase {
   }
 
   func testRecovery177() {
+    // rdar://38225184
```

Comment:
This isn't needed any more (I'm fine with just fixing that on main FWIW)

Reply:
Just to be clear.
Should it be removed or? 😁

Reply:
@bnbarham Just for clarification, this radar number has been carried over from the old parser tests because it was the motivation for the test case. I agree that it has very marginal value by now. I personally don’t have a strong opinion to keeping or removing it.

@kimdv just for your information: The contents of that radar are the same as https://github.com/apple/swift/issues/54086

Reply:
Fair enough, personally I'd much prefer a comment on what it's testing rather than a radar number (or the GitHub issue). For something with tonnes of details, sure, link out to the issue. For this though... 🤷 

Anyway, I know you didn't add it - it was mostly a drive by comment. No need to fix in this PR.